clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -189,7 +195,7 @@ public class AzureComputeClient extends AzureBaseClient {
     )
   }
 
-  ServiceResponse<Void> disableServerGroup(String resourceGroupName, String serverGroupName) {
+  ServiceResponse<Void> powerOffServerGroup(String resourceGroupName, String serverGroupName) {
```

Comment:
When we define that story, then we can implement the appropriate methods. Right now, they are not used and should be removed